pub-critic/src/app/components/footer/Footer.tsx

```diff
@@ -0,0 +1,27 @@
+import classes from "./Footer.module.scss";
+import instagram from "../../assets/instagram.svg";
+import facebook from "../../assets/fb.svg";
+import twitter from "../../assets/twitter.svg";
+import youtube from "../../assets/youtube.svg";
```

Comment:
set `"baseUrl": ".",` in `tsconfig.json` for using the absolute imports

Reply:
:white_check_mark: added baseUrl and paths